nixos/modules/services/misc/xmrig.nix

```diff
@@ -3,9 +3,16 @@
 
 let
   cfg = config.services.xmrig;
```

Comment:
Also, shouldn't `xmrig` itself be rebuilt with `WITH_CUDA=ON` and `WITH_NVML=ON` options?